presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -160,44 +160,66 @@ fun AgendaScreen(
                 }
 
                 items(agendaScreenState.agendaItems) { agendaItem ->
-                    AgendaCard(
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .height(120.dp)
-                            .clip(shape = RoundedCornerShape(22.dp)),
-                        title = agendaItem.title,
-                        subtitle = agendaItem.description,
-                        dateTimeInfo = agendaItem.toDisplayDateTime(),
-                        agendaCardType = when(agendaItem.agendaType) {
-                            AgendaType.EVENT -> AgendaCardType.EVENT
-                            AgendaType.TASK -> AgendaCardType.TASK
-                            AgendaType.REMINDER -> AgendaCardType.REMINDER
-                        },
-                        isAgendaCompleted = false
-                    ) {
-                        println("Event ${agendaItem.id} has been clicked")
-                        agendaScreenEvent(AgendaScreenEvent.OnAgendaItemClicked(agendaItem))
-                        agendaScreenEvent(AgendaScreenEvent.OnChangeShowEditAgendaItemDropdownStatus(shouldOpen = true))
-                    }
-
-                    /** Open, Edit, Delete Agenda Items */
-                    AgendaDropDownMenu(
-                        modifier = Modifier
-                            .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor)
-                            .align(Alignment.BottomEnd),
-                        shouldOpenDropdown = agendaScreenState.shouldOpenEditAgendaDropdown,
-                        onCloseDropdown = {
-                            agendaScreenEvent(
-                                AgendaScreenEvent.OnChangeShowEditAgendaItemDropdownStatus(shouldOpen = false))
-                        },
-                        listOfMenuItemId = AgendaMenuActionType.values().map { it.titleId },
-                        onSelectedOption = { item ->
-                            agendaScreenState.agendaItemClicked?.let { agendaItem ->
-                                onSelectedEditAgendaItemClicked(agendaItem, AgendaMenuActionType.values()[item])
-                                agendaScreenEvent(AgendaScreenEvent.OnChangeShowEditAgendaItemDropdownStatus(shouldOpen = false))
+                    Box(modifier = Modifier.fillMaxWidth()) {
```

Comment:
@philipplackner This didn't work. And I have also tried adding a box to the `optionsButton` and the dropdownmenu inside the `AgendaCardType`. Neither worked.

Reply:
Check the video regarding Context Menu Philipp has on his channel

Reply:
@MKiperszmid I have check that video which is displaying the dropdownmenu where the user clicked. However,  I just want to position under the optionButton. Please help check the code,

Reply:
You put the whole AgendaCard in a single box which doesn't do anything. You need to put the drop down in the box with the composable you want to anchor it to, such as:
```
Box {
    DropdownMenu(...)
    IconButton(...) { ... }
}
```